src/data/settings/languages.js

```diff
@@ -1,10 +1,78 @@
 // @flow
 
-const LANGUAGES_MAP: { [LanguageCode]: string } = {
-  en: 'English',
-  ko: 'Korean',
-  ja: 'Japanese',
-  zh: 'Chinese',
+export type LanguageCode = 'en' |
+  'ru' |
+  'ko' |
+  'ja' |
+  'zh-Hans' |
+  'zh-Hant' |
+  'es'
+
+export type LanguageInfo = {
+  title: string,
+  code: LanguageCode,
+  iconName: string,
+  isDisabled: boolean,
+}
+
+export type LanguagesMap = {
+  [code: LanguageCode]: LanguageInfo,
 }
 
-export default LANGUAGES_MAP
+const LANGUAGES_CONFIG: LanguageInfo[] = [
+  {
+    title: 'English',
+    code: 'en',
+    iconName: 'ic_english_24',
+    isDisabled: false,
+  },
+  {
+    title: 'Русский',
+    code: 'ru',
+    iconName: 'ic_russia_24',
+    isDisabled: false,
+  },
+  {
+    // TODO: translate to korean
+    title: 'Korean',
+    code: 'ko',
+    iconName: 'ic_korean_24',
+    isDisabled: false,
+  },
+  {
+    // TODO: translate to Chinese Simplified
+    title: 'Chinese Simplified',
+    code: 'zh-Hans',
+    iconName: 'ic_chinese_simplified_24',
+    isDisabled: false,
+  },
+  {
+    // TODO: translate to Chinese Traditional
+    title: 'Chinese Traditional',
+    code: 'zh-Hant',
+    iconName: 'ic_chinese_traditional_24',
+    isDisabled: false,
+  },
+  {
+    // TODO: translate to Japan
+    title: 'Japan',
+    code: 'ja',
+    iconName: 'ic_japan_24',
+    isDisabled: false,
+  },
+  {
+    // TODO: translate to Spanish
+    title: 'Spanish',
+    code: 'es',
+    iconName: 'ic_spain_24',
```

Comment:
I suggest to remove `ic_` & `_24` common parts

Reply:
but would be great if we will use computed iconName, like `ic_${code.toLowerCase()}_24`